README.md

```diff
@@ -2,8 +2,9 @@
 <!-- markdownlint-disable MD033 -->
 
 ![Build status](https://badge.buildkite.com/df9e995b3a5e4b0bebce8b432b0bf48b092fd261b7017b65c1.svg)
-[![License](https://img.shields.io/github/license/opstrace/opstrace)](https://github.com/opstrace/opstrace/blob/main/LICENSE)
+[![License](https://img.shields.io/github/license/opstrace/opstrace)](LICENSE)
```

Comment:
does this correctly link to our license still?

Reply:
Yes.  It just links to the license file on the particular branch, which I think is the more correct behavior.  It matches how we do it with the code of conduct below.